aspnetcore/fundamentals/localization/select-language-culture.md

```diff
@@ -142,6 +142,25 @@ In the preceding example, the order of `QueryStringRequestCultureProvider` and `
 
 As previously mentioned, add a custom provider via <xref:Microsoft.AspNetCore.Builder.RequestLocalizationOptionsExtensions.AddInitialRequestCultureProvider%2A> which sets the order to `0`, so this provider takes the precedence over the others.
 
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+## User override culture
+
+In previous versions, there was no way a user could override the culture settings. The culture settings were always set by the system culture settings. In ASP.NET Core 8.0, the [RequestLocalizationOptions.CultureInfoUseUserOverride](xref:Microsoft.AspNetCore.Builder.RequestLocalizationOptions.CultureInfoUseUserOverride) property allows the user to override the culture settings:
+
+```csharp
+    app.UseRequestLocalization(options =>
+    {
+        options.CultureInfoUseUserOverride = false;
+    });
+```
```

Comment:
@halter73  can you approve this to go into [What's new in ASP.NET Core 8.0](https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-8.0?view=aspnetcore-7.0)
Once you approve I'll have @hishamco  copy/paste this into What's new and do a pr there. I'll clean this up a bit for this document.